core/kernel/classes/class.Resource.php

```diff
@@ -569,18 +569,22 @@ public function duplicate($excludedProperties = [])
     }
 
     /**
-     * remove any assignation made to this resource, the uri is consequently
+     * Remove any assignation made to this resource, the uri is consequently
+     *
+     * @deprecated Use \oat\tao\model\resources\Repository\ResourceRepository::delete() instead
```

Comment:
- [ ] The repository should belong to Generis, since the Class resource belongs to generis